specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-01-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5113 @@
+          "description": "The sku of the workspace."
+        },
+        "systemData": {
+          "description": "System data",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      }
+    },
+    "Compute": {
+      "type": "object",
+      "description": "Machine Learning compute object.",
+      "discriminator": "computeType",
+      "properties": {
+        "computeType": {
+          "description": "The type of compute",
+          "$ref": "#/definitions/ComputeType"
+        },
+        "computeLocation": {
+          "description": "Location for the underlying compute",
+          "type": "string"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "The provision state of the cluster. Valid values are Unknown, Updating, Provisioning, Succeeded, and Failed.",
+          "enum": [
+            "Unknown",
+            "Updating",
+            "Creating",
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the Machine Learning compute."
+        },
+        "createdOn": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time at which the compute was created."
+        },
+        "modifiedOn": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time at which the compute was last modified."
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "ARM resource id of the underlying compute"
+        },
+        "provisioningErrors": {
+          "type": "array",
+          "description": "Errors during provisioning",
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+          },
+          "readOnly": true
+        },
+        "isAttachedCompute": {
+          "type": "boolean",
+          "description": "Indicating whether the compute was provisioned by user and brought from outside if true, or machine learning service provisioned it if false.",
+          "readOnly": true
+        },
+        "disableLocalAuth": {
+          "type": "boolean",
+          "description": "Opt-out of local authentication and ensure customers can use only MSI and AAD exclusively for authentication."
+        }
+      },
+      "required": [
+        "computeType"
+      ]
+    },
+    "AKS": {
+      "description": "A Machine Learning compute based on AKS.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Compute"
+        },
+        {
```

Comment:
Please do not use anonymous schema here, which usually generates bad SDK code.
Also please do not use multi-inheritance, which usually generates bad SDK code as well